scripts/vm/network/security_group.py

```diff
@@ -149,7 +149,7 @@ def split_ips_by_family(ips):
 
 
 def get_bridge_physdev(brname):
-    physdev = execute("bridge -o link show | awk '/master %s / && !/^[0-9]+: vnet/ {print $2}'" % brname)
+    physdev = execute("bridge -o link show | awk '/master %s / && !/^[0-9]+: vnet/ {print $2}' | head -1" % brname)
```

Comment:
Can you rewrite this to use `iproute2`?

Reply:
Excerpt from commit message:

>     In addition use the 'bridge' command instead of 'brctl'. 'bridge' is part of the iproute2 utils just like 'ip' and should be considered as the new default.